bankbarcode/cuaderno57.py

```diff
@@ -83,7 +110,8 @@ class Recibo507(Recibo):
         Autoservicio, V2001)
     """
 
-    def __init__(self, entity, suffix, ref, notice, amount):
+    def __init__(self, entity=None, suffix=None, ref=None, notice=None,
```

Comment:
If I remember well this arguments are required, so if you named it we need to add a constrain.